File: python/LC-26/LC-26.py
from ast import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0
        count = 0 

        for i in nums:
            if i not in nums[:count]:
                nums[count] = i
                count+=1
        return count

File: golang/LC-26/LC-26.go
func removeDuplicates(nums []int) int {
	if len(nums) < 1 {
		return len(nums)
	}
	counter := 0
	for i := 0; i < len(nums); i++ {
		if nums[counter] != nums[i] {
			counter++
			nums[counter] = nums[i]
		}
	}
	return counter + 1
}